learn-accessibility-by-building-a-website-for-people-with-disabilities/css/style560.css

```diff
@@ -0,0 +1,147 @@
+body {
+  padding: 0;
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+  outline: none;
+  /*
+    To remove default browser styles and provide a unified look, web developers
+    and designers often set `outline` to `none` on selectable elements.
+
+    However, this is a disaster for accessibility, because it makes the
+    currently focused element invisible, making keyboard navigation almost
+    impossible.
+
+    If other styles (such as a `box-shadow`) are provided instead,
+    accessibility can be restored, but care must be taken to ensure that
+    focused elements always have _some_ distinguishing style.
+
+    In this case, we can fix the problem by simply removing the rule.
```

Comment:
In this case, we can fix the problem by simply removing the `outline: none;` declaration.